.github/workflows/build.yaml

```diff
@@ -51,3 +51,25 @@ jobs:
     - name: Run Tests
       run: |
         make test
+  docker-build:
```

Comment:
Since this build is currently so time consuming we should try to limit when it runs as much as feasible. At minimum, we shouldn't run this when only control-plane components are changed. At best, we should identify the _exact_ circumstances under which this _must_ be run and configure the runs as surgically as possible.